applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -143,17 +146,45 @@ static struct bt_audio_stream_ops stream_ops = { .sent = stream_sent_cb,
 						 .started = stream_started_cb,
 						 .stopped = stream_stopped_cb };
 
+static void public_broadcast_features_set(uint8_t *features)
+{
+	int freq = bt_codec_cfg_get_freq(&lc3_preset.codec);
+
+	if (features == NULL) {
+		LOG_ERR("No pointer to features");
+		return;
+	}
+
+	if (IS_ENABLED(CONFIG_BT_AUDIO_BROADCAST_ENCRYPTED)) {
+		*features |= 0x01;
+	}
+
+	if (freq == STANDARD_QUALITY_16KHZ || freq == STANDARD_QUALITY_24KHZ) {
+		*features |= 0x02;
+	} else if (freq == HIGH_QUALITY_48KHZ) {
+		*features |= 0x04;
+	}
```

Comment:
Should have a print or something if freq is none of the above?

Reply:
Not sure what the behaviour should be though. We could just print a warning of course, but then no bit is set, and then I guess it's not Auracast. Or Do we set standard quality for all frequencies below 24kHz and high quality if it's larger?

I'll add a warning for now